src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSService.kt

```diff
@@ -173,6 +194,26 @@ class SNSAppointmentService(
           mapOf("serviceUserCRN" to referral.serviceUserCRN, "referralId" to referral.id)
         )
 
+        snsPublisher.publish(referral.id, appointment.appointmentFeedbackSubmittedBy!!, snsEvent)
+      }
+      AppointmentEventType.SESSION_FEEDBACK_RECORDED -> {
```

Comment:
this duplication isn't nice ... is this a hang over from the recent appointment changes? is there a way to factor this out?

Reply:
There is much duplication between AppointmentEvent and ActionPlanAppointmentEvent. I think the latter needs to be migrated to using the later Appointment design. I have raised a task for it. 
https://trello.com/c/g7YJYkRI/154-are-actionplanappointmentevents-still-appropriate